### DISCUSSION TITLE
how to use "vercel deploy --prebuilt" with nitro

### DISCUSSION THREAD

**Discussion by sandys** [nitrojs/nitro#1754] [General]
hi,
we build our vercel serverless and edge apis locally and deploy them using "vercel deploy --prebuilt". This is a very supported workflow by Vercel - https://vercel.com/guides/deploying-locally-built-nextjs

does anyone have any idea on how to do this with nitro ? as long as i have a prebuilt vercel build output api compatible output, i will be able to run "vercel deploy --prebuilt" without any problems.

**Comment by pi0** [comment]
Moved to issue https://github.com/unjs/nitro/issues/1765
